lib/src/se/llbit/chunky/ui/IntegerTextField.java

```diff
@@ -6,7 +6,19 @@
 public class IntegerTextField extends NumericTextField<IntegerProperty> {
 
   public IntegerTextField() {
-    super(new SimpleIntegerProperty());
-    this.getConverter().setParseIntegerOnly(true);
+    this(0);
```

Comment:
Does this break any other usages?

Reply:
You had me worried for a second, but it is the same code as before, only 4 lines lower :)